tests/D2L.CodeStyle.Analyzers.Test/Specs/ConstantAnalyzer.cs

```diff
@@ -0,0 +1,69 @@
+﻿// analyzer: D2L.CodeStyle.Analyzers.Helpers.ConstantAnalyzer
+
+using D2L.CodeStyle.Annotations;
+
+#region Relevant Types
+namespace D2L.CodeStyle.Annotations

```

Comment:
Don't know why you editor isn't applying it properly, but, some spacing issues here:

```csharp
namespace Foo {
	public class Bar {
		public void Method( object parameter ) {
			Method( "foo" );
		}
	}
}
```

Reply:
I think spacing is good now.